src/promptflow/tests/executor/unittests/executor/test_exceptions.py

```diff
@@ -12,3 +12,4 @@ def test_exception_message(self):
         )
 
         assert ex.message == "Test exception message with parameters: test_param, <param1>."
+        assert None not in ex.message_parameters
```

Comment:
We should put this line in a new test case named `test_get_arguments_from_message_format`.

Reply:
Good suggestion. Since the code is very simple now, may improve it in future.